components/ws-manager-bridge/src/workspace-instance-controller.ts

```diff
@@ -144,7 +137,7 @@ export class WorkspaceInstanceControllerImpl implements WorkspaceInstanceControl
         const span = TraceContext.startSpan("controlNonStoppedWSManagerManagedInstances", parentCtx);
         const ctx = { span };
         try {
-            log.debug("Controlling ws-manager managed instances...", { workspaceClusterName });
+            log.debug("Controlling ws-manager managed instances...");
```

Comment:
The clusterName was only used in logs

Reply:
But that's also very important part of the logs :laughing: I used it multiple times in the past to rule out errors cases.

Why remove it in the first place? :thinking: Also, it's independent of the change, right?

Reply:
Technically, the value that's supplied is the value of https://github.com/gitpod-io/gitpod/pull/16876/files#diff-e7027a2857b316687949201e4fbd013e2bea4d59b0cce596c4f6fa997bdff2e3L51

Which is always the current application cluster. The naming in this method is misleading, as it's not the workspaceClusterName, but it's actually the application cluster. And given we only have one, it's always the same value.

Reply:
I think somebody of us is confusing something: I see this method being called [here](https://github.com/gitpod-io/gitpod/blob/495dab06d756610a60256f732badf352490476b5/components/ws-manager-bridge/src/workspace-instance-controller.ts#L91-L97), using the workspaceClusterName from [here](https://github.com/gitpod-io/gitpod/blob/495dab06d756610a60256f732badf352490476b5/components/ws-manager-bridge/src/workspace-instance-controller.ts#L72)? :thinkin:

Reply:
:dart: